perllib/FixMyStreet/Cobrand/Westminster.pm

```diff
@@ -153,4 +153,13 @@ sub categories_restriction {
         }, { join => 'body' });
 }
 
+sub updates_restriction {
+    my $self = shift;
+
+    # Westminster don't want any fms.com updates shown on their cobrand.
+    return $self->next::method(@_)->search({
+        "me.cobrand" => 'westminster'
```

Comment:
All the Open311-imported updates do not have a cobrand at all, which means this will hide all of them. Given any .com created ones will have a cobrand, and you can't create westminster updates, I guess checking for blank string here instead would do?

Reply:
Oops! I've updated it to specifically exclude only `fixmystreet` cobrand updates, just in case we enable updates on Westminster in the future and wonder why they're not appearing :)